server/db/db.js

```diff
@@ -0,0 +1,51 @@
+const { open } = require('sqlite');
+const sqlite3 = require('sqlite3').verbose();
+let db;
+
+ (async () => {
+  // open the database
+    db = await open({
+    filename: './server/db/app.db',
+    driver: sqlite3.Database
+  })
+  // Create a table just for once
+  db.run(`CREATE TABLE IF NOT EXISTS user (
+    user_id INTEGER PRIMARY KEY,
+    user_name STRING NOT NULL UNIQUE,
+    password STRING NOT NULL)`
+  );
+})()
+
+async function checkIn (name) {
+  try {
+    const result = await db.get(`SELECT user_id, user_name, password FROM user WHERE user_name='${name}'`);
+    return result
+  } catch (err) {
+     return(err);
```

Comment:
btw, can you also console.error all the errors on this page before returning them?